packages/contracts-bedrock/src/L2/L1BlockInterop.sol

```diff
@@ -2,108 +2,103 @@
 pragma solidity 0.8.15;
 
 import { L1Block } from "src/L2/L1Block.sol";
+import { EnumerableSetLib } from "@solady/utils/EnumerableSetLib.sol";
```

Comment:
We should consider other implementations of this library given the lack of the sentinel value that it cannot hold onto as part of its set